README.md

```diff
@@ -79,7 +79,33 @@ Install the following basic prerequisites:
 * Docker 18.09 or later (and ensure its daemon is running).
 * Enable [BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/) for Docker.
 
-Change into the directory with ORT's source code and run `docker build -t ort .`.
+Change into the directory with ORT's source code and run: 
+```
+docker build -t ort .
+```
+
+Tool versions can be changed passing as args like **--build-arg TOOL_ENV=VERSION**
+This is the list of tools you can change version and the defaults
+
+Tool Env Var | Default Version
+------------ | ---------------
+ANDROID_SDK_VERSION | 6858069
+BOWER_VERSION | 1.8.8
+CONAN_VERSION | 1.38.0
+GO_DEP_VERSION | 0.5.4
+GO_VERSION | 1.16.6
+HASKELL_STACK_VERSION | 2.1.3
+NPM_VERSION | 6.14.2
+PYTHON_VERSION | 3.8.11
+SBT_VERSION | 1.3.8
+SCANCODE_VERSION | 21.7.30
```

Comment:
This kind off documentation out dates pretty fast ;)

Reply:
@maxhbr I accept suggestion on how can handle it

Reply:
Instead of listing versions here, I would suggest to point to ARGs in the Dockerfile itself